Source/WebCore/testing/Internals.cpp

```diff
@@ -7462,27 +7462,22 @@ AccessibilityObject* Internals::axObjectForElement(Element& element) const
 
 String Internals::getComputedLabel(Element& element) const
 {
-    if (auto* axObject = axObjectForElement(element))
-        return axObject->computedLabel();
-    return ""_s;
-}
+    // Force a layout. If we don't, and this request has come in before the render tree was built,
+    // the accessibility object for this element will not be created (because it doesn't yet have its renderer).
+    if (RefPtr document = contextDocument())
+        document->updateLayoutIgnorePendingStylesheets();
 
-String Internals::getComputedRole(Element& element) const
-{
-    if (auto* axObject = axObjectForElement(element))
-        return axObject->computedRoleString();
-    return ""_s;
+    RefPtr axObject = axObjectForElement(element);
```

Comment:
are role and label the only things that are impacted?

Should we move 
if (RefPtr document = contextDocument())
        document->updateLayoutIgnorePendingStylesheets();

into a standard function that can be called by any methods that need to do this

Reply:
> Should we move ... into a standard function that can be called by any methods that need to do this

Fixed!

> are role and label the only things that are impacted?

Yes, these are the only two properties we need for the WPTs at this point.